plugins/web/opentelemetry-plugin-user-interaction/test/userInteraction.test.ts

```diff
@@ -40,6 +40,21 @@ import {
 const FILE_URL =
   'https://raw.githubusercontent.com/open-telemetry/opentelemetry-js/master/package.json';
 
+// this is needed until this is merged and released
+// https://github.com/open-telemetry/opentelemetry-js/pull/1209
```

Comment:
Do you think we need to open issue to track this later?

Reply:
1209 is merged and released now actually

Reply:
https://github.com/open-telemetry/opentelemetry-js/releases/tag/v0.9.0